### ISSUE TITLE
Relativistic Au Calculation

### ISSUE THREAD

**Issue by wmatter** [shankar1729/jdftx#320]
I have been trying to do a DFT+U calculation for solid Au using the pseudopotential
https://pseudopotentials.quantum-espresso.org/upf_files/Au.rel-pbesol-spn-rrkjus_psl.1.0.0.UPF

I have tried using a variety of Hubbard U parameters, including U=-0.0583 Hartrees from linear response and U=2.5 eV from optimizing the energy of the d-band to match literature. I am running into an issue where the band structure (attached) intersects the Fermi level to the right of the L symmetry point, meaning that the Fermi surface doesn’t intersect the Bragg plane as would be expected for the noble metals. This issue doesn’t seem to resolve no matter what I have been trying (input file attached).

![bandstruct](https://github.com/shankar1729/jdftx/assets/32248694/f41c59c6-cbf0-4ef0-9803-9ff31b84c4bb)

Ultimately we are aiming to produce an accurate DOS with the correct d-band energy and with a relativistic pseudopotential and XC functional. Any suggestions would be greatly appreciated. Many thanks.

[Au.in.txt](https://github.com/shankar1729/jdftx/files/14795194/Au.in.txt)

**Comment by shankar1729** [comment]
I could reproduce your issue with the PSlib pseudopotential in the attached results. I did adjust the plane wave cutoffs to half of what you had since the recommendations in the UPF are in Ry, while JDFTx uses Eh. I also used U = 0.075 Eh that we had fit to ARPES in an old work, but that's not too far from the values you tried. All these don't make a difference though, and the bandstructures are bad.

Seems like that PSlib pseudopotential is bad. I also attach results for the [PBEsol FR NC pseudopotential from DOJO](http://www.pseudo-dojo.org/pseudos/nc-fr-04_pbesol_stringent/Au.upf.gz?download), with everything else the same, and it gives a reasonable looking bandstructure.

[Tests.zip](https://github.com/shankar1729/jdftx/files/14795746/Tests.zip)

Best,
Shankar
